apps/emqx_connector/src/emqx_connector_redis.erl

```diff
@@ -52,6 +37,21 @@
 
 -export([cmd/3]).
 
+
+%% redis server(s) parse
+-type server_raw() :: tuple().
+-type servers_raw() :: [server_raw()].
```

Comment:
try to use primitive types and add use converter + validator for the field.

https://github.com/emqx/hocon/blob/0cf2a9e1173aad39bdbd35445f74a755e4707f20/src/hocon_schema.erl#L80-L81

primitive types such as `string()` is more readable in the generated document.
the parsing rules can be explained in `desc` schema field.